src/setuptools_scm/version.py

```diff
@@ -20,6 +20,38 @@
 SEMVER_LEN = 3
 
 
+class NonNormalizedVersion:
+    """A non-normalizing version handler.
+
+    You can use this class to disable all version verification and normalization.
+    For example you can use this to avoid git release candidate version tags
+    ("1.0.0-rc1") to be normalized to "1.0.0rc1". Only use this if you fully
+    trust the version tags.
+    """
+
+    def __init__(self, tag):
```

Comment:
i consider it mandatory to ensure the version is a valid python version number here, i would recommend to subclass Version and storing the non-normalized value on  for repr/str use

Reply:
ok